TShockAPI/Commands.cs

```diff
@@ -5408,12 +5475,12 @@ private static void Rocket(CommandArgs args)
 		{
 			if (args.Parameters.Count != 1)
 			{
-				args.Player.SendErrorMessage("Invalid syntax! Proper syntax: {0}rocket <player>", Specifier);
+				args.Player.SendErrorMessage($"Invalid syntax! Proper syntax: {Specifier}rocket <player>");
 				return;
 			}
 			var players = TSPlayer.FindByNameOrID(args.Parameters[0]);
 			if (players.Count == 0)
-				args.Player.SendErrorMessage("Invalid player!");
+				args.Player.SendErrorMessage($"{players} is not a valid player!");
```

Comment:
```suggestion
				args.Player.SendErrorMessage($"{args.Parameters[0]} is not a valid player!");
```
`players` will be an empty list here.